WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Print Shipping Label/PrintShippingLabelViewController.swift

```diff
@@ -107,6 +107,16 @@ private extension PrintShippingLabelViewController {
 private extension PrintShippingLabelViewController {
     func configureNavigationBar() {
         navigationItem.title = Localization.navigationBarTitle(labelCount: viewModel.shippingLabels.count)
+        navigationItem.leftBarButtonItem = UIBarButtonItem(title: Localization.closeButtonTitle,
+                                                           style: .plain,
+                                                           target: self,
+                                                           action: #selector(closeButtonTapped))
+        isModalInPresentation = true
+    }
+
+    @objc func closeButtonTapped() {
+        onAction?(.saveLabelForLater)
```

Comment:
nit: can it just call `saveLabelForLater` without the `presentingViewController?.dismiss(animated: true)` line below?
```suggestion
        saveLabelForLater()
```

Reply:
`dismiss` is required, because `saveLabelForLater` isn't available/relevant when you're reprinting a label. I'll check on the function call, I didn't spot that one so thank you for pointing it out

Reply:
Done in e3c2a94